### PR TITLE
Chrisdev login 1202470074886586

### PR DESCRIPTION
**Pull request by chriscodev**
Disable Login button once authentication process has started

ASANA: [1202470074886586](https://app.asana.com/0/1202417486432147/1202470074886586)

### CODE REVIEW

**Comment by triveni-codev** [comment]
Spinner design and animation considerations confirmed.

However I have the following concerns.
1. Resetting concern: The Login button would go to infinite loading when the login form is invalid (Example: Username and password is empty and then Login button is pressed) 

Could maybe use a toggler line for disabledLogin that resembles somewhat like this?

image

2. On successful authentication, There is a significant hang time where the spinning animation in the login button stops prematurely before the page transitions to landing page.
TLDR: Ensure that the spinner animation(on successful login) does not stop prematurely.

Login revcomNum2

3. Loophole concern: In relation to review comment no. 2, I was able to send login request twice because the login button got re-enabled prematurely. If review comment #2 is fixed, this would be automatically fixed I suppose.

**Comment by triveni-codev** [comment]
4 files affected for this change is confirmed. Functionality confirmed. Review comments/concerns have been addressed.

**Comment by triveni-codev** [comment]
Resynced changes from master to only reveal 4 affected files.
